src/tokio/io.rs

```diff
@@ -160,16 +160,16 @@ where
                 let mut result = Err(e);
 
                 for (i, duration) in (options.retries_to_attempt_fn)().enumerate() {
-                    let reconnect_num = i + 1;
+                    let _reconnect_num = i + 1;
```

Comment:
I do not think these  need to be changed to lead with _

Reply:
I changed it to that to avoid rustc warnings for unused variables when compiling without logging. Those variables are only used in the log statements. Up to you how you want to handle the warnings, I can remove the underscores if you want

Reply:
We shouldn't put the burden on the code here, that should be something covered in the macros itself, perhaps with an allow unused macro in the logging macros you created.

Reply:
Should be resolve in [6887535](https://github.com/craftytrickster/stubborn-io/pull/26/commits/68875359529f1a253bed09f9e20382058eb9789e)